modules/extensions/src/data-filter/data-filter.js

```diff
@@ -40,12 +43,12 @@ const DATA_TYPE_FROM_SIZE = {
 };
 
 export default class DataFilterExtension extends LayerExtension {
-  constructor({filterSize = 1, fp64 = false} = {}) {
+  constructor({filterSize = 1, fp64 = false, countItems = false} = {}) {
     if (!DATA_TYPE_FROM_SIZE[filterSize]) {
       throw new Error('filterSize out of range');
     }
 
-    super({filterSize, fp64});
+    super({filterSize, fp64, countItems});
```

Comment:
Total Nit: It is a bit weird to write it like this, if you are just passing along a props object. It makes it look as if `LayerExtension` knows about those fields.